src/main/java/org/jenkinsci/plugins/docker/commons/tools/DockerToolInstaller.java

```diff
@@ -0,0 +1,118 @@
+package org.jenkinsci.plugins.docker.commons.tools;
```

Comment:
IANAL but my impression was that at least all `*.java` should have it. Anyway, resolved.